.babelrc

```diff
@@ -1,9 +1,8 @@
 {
 	"presets": [
-		[ "latest", {
-			"es2015": {
-				"modules": false
```

Comment:
This was important to have in order to achieve [Webpack tree shaking](https://webpack.js.org/guides/tree-shaking/#components/sidebar/sidebar.jsx).